images/sources-readme.txt

```diff
@@ -0,0 +1,7 @@
+
+Due to the way this image is built, the source container for the image itself
```

Comment:
Maybe "this image" should be more specific. Or maybe this entire readme text could be added in comments in the docker file. Wdyt?

Reply:
Yeah, potentially. You can also make it more explicit by including in the Dockerfile for the image. Or if this text doesn't end up being much, we can just only put it in the Dockerfile as well.